build/build.proj

```diff
@@ -103,5 +104,47 @@
     <Error Text="There were test failures, for full log see %(HtmlTestFile.FullPath)" Condition="$(ExitCode) != 0" />
 
   </Target>
- 
+
+  <Target Name="TestIntegration" Condition="'$(RunIntegrationTests)' == 'true'">
+    <PropertyGroup>
+      <DownloadRoslynVsixesExe>$(NUGET_PACKAGES)\roslyntools.downloadroslynvsixes\0.3.2-beta\tools\DownloadRoslynVsixes.exe</DownloadRoslynVsixesExe>
```

Comment:
We done this for some of the Nuget packages already [here](https://github.com/dotnet/project-system/blob/master/build/Targets/VSL.Versions.targets#L21)